games/Duck_Hunt.js

```diff
@@ -0,0 +1,279 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Thanks for PR! Wdym with this?

Reply:
The code is licensed under the BSD 3 clause. Probably need to add the text tho

Reply:
updated to include BSD 3-clause license text. BSD3 shouldn't prevent hackclub from merging this code. Info at [opensource.org](https://opensource.org/license/BSD-3-clause/)

Reply:
Ok! I'll check:)

Reply:
The entire project is under the [MIT License](https://github.com/hackclub/sprig/blob/main/LICENSE), so it would be a license conflict, and I particularly don't think it's necessary to put a specific license for a specific game

Reply:
I don't think that they conflict. If you want to use the MIT license you can just include the copyright notice on the BSD-3 code or just remove it. I think it's better to explicitly license code instead of assuming it falls under another.

Reply:
Then remove and add the one from MIT, so everything is padronized

Reply:
dual-licensed

Reply:
Does your license allow this? e.g. Creative Commons Licensing is not allowed to dual-license

Reply:
There is no restriction in either license against dual-licensing, and it would be an either or MIT / BSD.

Reply:
do you have a link of this license?

Reply:
How do you mean?